tests/phpunit/elementor/test-elements.php

```diff
@@ -106,4 +108,15 @@ public function test_controlsDefaultData() {
 			}
 		}
 	}
+
+	private function removeCommasFromSelectorBrackets( $selector ) {
+		$isContainCommaRegex = '/\(.*,+.*\)/';
+		preg_match( $isContainCommaRegex, $selector, $matches );
```

Comment:
it doesn't really matter since i clean the comma and return a clean string , so even if we have 
asdasd ( .a , .b ) , asdasd (.c , .d )
it will capture  from ( to ) and will return a clean string

Reply:
hmm  ....
and this is a bug :(